Translations/translation_EN.json

```diff
@@ -257,14 +257,14 @@
 				"Power",
 				"pulse"
 			],
-			"desc": "Intensity of power of keep-awake-pulse (watt)"
+			"desc": "Power intensity of keep-awake-pulse (watt)"
 		},
 		"HallEffSensitivity": {
 			"text2": [
 				"Hall sensor",
 				"sensitivity"
 			],
-			"desc": "Sensitivity of the Hall effect sensor to trigger sleep mode (0=off | 1=least sensitive | ... | 9=most sensitive)"
+			"desc": "Sensitivity to magnets (0=off | 1=least sensitive | ... | 9=most sensitive)"

```

Comment:
Same as above in **`Settings.md`**.
Hall effect sensors are built to detect changes in magnetic fields. What else should they be sensitive to if not magnets?